### PR TITLE
update auto can-i error message

### PR DESCRIPTION
**Pull request by bobsongplus**
#### What type of PR is this?
/kind bug
/kind cleanup

#### What this PR does / why we need it:
NONE
#### Which issue(s) this PR fixes:

Fixes #https://github.com/kubernetes/kubernetes/issues/110265

#### Special notes for your reviewer:
NONE
#### Does this PR introduce a user-facing change?
NONE

### CODE REVIEW

**Comment by sftim** [comment]
@TinySong, please check https://github.com/kubernetes/kubernetes/pull/110267#issuecomment-1140303570 for hints about how to fix the issue with the release note.

The string `NONE` needs to match exactly; `none` is not equivalent.

**Comment by TinySong** [comment]
I have read [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md), and I think there no need release note, remove lease note labels.

/release-note-none

cc @sftim @deads2k @enj

File: staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go [hunk 1]
```diff
@@ -174,7 +174,9 @@ func (o *CanIOptions) Complete(f cmdutil.Factory, args []string) error {
 				o.ResourceName = resourceTokens[1]
 			}
 		default:
-			return errors.New("you must specify two or three arguments: verb, resource, and optional resourceName")
+			errString := "you must specify two arguments: verb  resource or verb resource/resourceName."
```
Comment (rikatz) [inline staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go:177]: nit: remove the duplicate space between `verb` and `resource`

**Comment by rikatz** [comment]
/ok-to-test
/lgtm
There's one minor nit, otherwise looks good to me

File: staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go (hunk 1 above)
Comment (TinySong) [reply staging/src/k8s.io/kubectl/pkg/cmd/auth/cani.go]: done

**Review by rikatz — APPROVED** [review]
/lgtm
Thanks

**Comment by enj** [comment]
/lgtm

@deads2k PTAL

**Pull request merged**